src/MooLite/plugins/WhisperManager/WhisperManagerPlugin.ts

```diff
@@ -0,0 +1,138 @@
+import { MooLitePlugin } from "src/MooLite/core/plugins/MooLitePlugin";
+import { MooLiteTab } from "src/MooLite/core/plugins/MooLiteTab";
+import { markRaw } from "vue";
+import WhisperManagerPluginDisplay from "src/MooLite/plugins/WhisperManager/WhisperManagerPluginDisplay.vue";
+import { ChatMessage } from "src/MooLite/core/chat/ChatMessage";
+import { ChatChannelTypeHrid } from "src/MooLite/core/chat/ChatChannelTypeHrid";
+import { ConversationMessage } from "./ConversationMessage";
+import { conversationData } from "src/MooLite/plugins/WhisperManager/DevConstants";
+import { Game } from "src/MooLite/core/Game";
+import { Conversation } from "src/MooLite/plugins/WhisperManager/Conversation";
+import { PluginConfig } from "src/MooLite/core/plugins/config/PluginConfig";
+import { PluginConfigType } from "src/MooLite/core/plugins/config/PluginConfigType";
+
+export class WhisperManagerPlugin extends MooLitePlugin {
+    name: string = "Whisper Manager";
+    key = "whisper-manager";
+    description: string = "A plugin to help manage all of your conversations!";
+
+    _messages: ChatMessage[] = [];
+    _conversations: Record<string, Conversation> = {};
+    _activeConversationName: string = "";
+
+    config: PluginConfig[] = [
+        {
+            type: PluginConfigType.CheckBox,
+            name: "Hide System Messages",
+            description: "Hide system messages from the conversation list",
+            key: "hide-system-messages",
+            value: false,
+        },
+        {
+            type: PluginConfigType.CheckBox,
+            name: "Hide Mod Messages",
+            description: "Hide mod messages from the conversation list",
+            key: "hide-mod-messages",
+            value: false,
+        },
+    ];
+
+    tab: MooLiteTab = {
+        icon: "💬",
+        pluginName: this.name,
+        componentName: "WhisperManagerPluginDisplay",
+        component: markRaw(WhisperManagerPluginDisplay),
+        width: "w-80",
+    };
+
+    initialize(game: Game): void {
+        super.initialize(game);
+        console.log("WhisperManagerPlugin initialized");
```

Comment:
Remove debug print, I think you can remove this entire `initialize()`

Reply:
Yep.